modules/network/variables.tf

```diff
@@ -0,0 +1,8 @@
+variable "base_cidr_block" {
+  type = string
+}
+
+variable "env_name" {
+  type        = string
+  description = "Name to be used for the current environment, ie staging or production"
+}
```

Comment:
We usually use `namespace` for this purpose on other projects. Do you consider using it as well?


```suggestion
variable "namespace" {
  type        = string
  description = "Name to be used for the current environment, ie example-web-staging or example-web-production"
}
```

Reply:
@hoangmirs So this includes environment and the name of the app? Alright I'll change this and according to your comments below

Reply:
I changed to namespace but I will need the environment name anyway for the image tag for `staging` and `prod` as it's tagged that during ci/cd workflow when uploading ECR

Reply:
unless I use the `split` function and grab last element